test/common/core/Client.ts

```diff
@@ -46,7 +47,7 @@ describe("Client.ts", () => {
 			};
 			const client: Client = Client.initWithMiddleware(options);
 			assert.isTrue(client instanceof Client);
-			assert.isDefined(client["httpClient"]);
+			assert.isDefined(["httpClient"]);
```

Comment:
how does this work?

Reply:
Thanks for catching this! Corrected this.